File: annotations/src/main/java/com/kazakago/preferhythm/PrefKeyName.java
package com.kazakago.preferhythm;

import java.lang.annotation.ElementType;
import java.lang.annotation.Target;

/**
 * Annotation of Preferences ModelField.
 * this can change key name.
 *
 * Created by KazaKago on 2017/03/08.
 */
@Target(ElementType.FIELD)
public @interface PrefKeyName {
    String value();
}

File: annotations/src/main/java/com/kazakago/preferhythm/PrefKeyName.kt
package com.kazakago.preferhythm

/**
 * Annotation of Preferences ModelField.
 * this can change key name.

 * Created by KazaKago on 2017/03/08.
 */
@Target(AnnotationTarget.FIELD)
annotation class PrefKeyName(val value: String)